mod_reforged/hooks/contracts/contracts/deliver_item_contract.nut

```diff
@@ -0,0 +1,14 @@
+::mods_hookExactClass("contracts/contracts/deliver_item_contract", function(o) {
+    local onPrepareVariables = o.onPrepareVariables;
+    o.onPrepareVariables = function( _vars )
+    {
+        onPrepareVariables(_vars);
+        foreach (var in _vars)
+        {
+            if (var[0] != "days") continue;
+            local seconds = ::Reforged.Text.getSecondsRequiredToTravel(this.m.Flags.get("Distance"), ::Const.World.MovementSettings.Speed, true);
```

Comment:
Personally I don't like showing "days + hours" as it is "too exact" and doesn't fit the atmosphere of the game. What I would prefer is that you round it to the nearest number of half days. So if it takes closer to 1 day, you say 1 day. If it takes closer to 1.5 you say 1 and a half days. If it takes closer to 2 days you say 2 days. And so on. That is still "accurate enough" information for the player to compare, and is immersive in the game's universe.